### PR TITLE
fix MT vaccine completed

### PR DESCRIPTION
**Pull request by smcclure17**
We were pulling from a strictly "2 dose" column that did not include j&j doses. Updated to pull correct `total_vaccine_completed` data and added `total_vaccine_doses_administered` data.

### CODE REVIEW

File: can_tools/scrapers/official/MT/mt_vaccinations.py [hunk 1]
```diff
@@ -18,11 +18,16 @@ class MontanaCountyVaccine(ArcGIS):
             measurement="cumulative",
             unit="people",
         ),
-        "Dose_2": CMU(
+        "Fully_Vaxed": CMU(
```
Comment (ghop02) [inline can_tools/scrapers/official/MT/mt_vaccinations.py:21]: I bet the MT scraper was a bit old, but could use update the values here to use the variable constants?

**Review by ghop02 — APPROVED** [review]
LGTM! Do you think we'll be able to rerun this on old data? (I believe there's a command for that) - I can also follow up on that part!

File: can_tools/scrapers/official/MT/mt_vaccinations.py (hunk 1 above)
Comment (smcclure17) [reply can_tools/scrapers/official/MT/mt_vaccinations.py:21]: @ghop02 yeah I can add the variables!

I don't know much (anything) about re-running for the old data--but I can look into it. Do you know where/how that might be done?

File: can_tools/scrapers/official/MT/mt_vaccinations.py [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import pandas as pd
 import us
 
-from can_tools.scrapers import CMU
+from can_tools.scrapers import variables as v
```
Comment (ghop02) [inline can_tools/scrapers/official/MT/mt_vaccinations.py:4]: In general we should try to shy away from single letter variables - does `variables` need to be aliased?

Comment (smcclure17) [reply can_tools/scrapers/official/MT/mt_vaccinations.py:4]: ah ok, no it doesn't--was just for ease of typing haha, I can remove that

Comment (ghop02) [reply can_tools/scrapers/official/MT/mt_vaccinations.py:4]: thanks!

Comment (smcclure17) [reply can_tools/scrapers/official/MT/mt_vaccinations.py]: should be fixed!

**Review by ghop02 — APPROVED** [review]
LGTM! Thanks for hopping on this!

**Comment by smcclure17** [comment]
i'll wait for the tests and then I'll merge

**Comment by ghop02** [comment]
ooo tests take a very long time - we should really figure out a better way to test - it looks like we essentially run the pipeline locally, which isn't super helpful for unit tests.  Could you just run the tests locally for this scraper and confirm they pass? (I'd just like to get it in sooner rather than later)

**Comment by smcclure17** [comment]
yep they pass, i'll go ahead and merge!

Yeah--they do take a loong time. I think it's mainly b/c the demographic scrapers have to make a lot of requests (often one per county) so they can each take anywhere from 2-10 minutes. 

We run each scraper to make sure that all the locations/demographic values/variables are valid (exist within the `bootstrap_data` files) but that does mean it takes a lot of time. I can put some thought into a better way of doing this

**Pull request merged**